opsmngr/all_clusters_test.go

```diff
@@ -0,0 +1,227 @@
+package opsmngr
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+
+	"github.com/go-test/deep"
+)
+
+func TestAllClusters_List(t *testing.T) {
+	setup()
+
+	defer teardown()
+
+	var trueValue = true
+	var falseValue = false
+
+	mux.HandleFunc("/clusters", func(w http.ResponseWriter, r *http.Request) {
+		_, _ = fmt.Fprint(w, `{
+				 "results": [
+				{
+				  "groupName": "AtlasGroup1",
+				  "orgName": "TestAtlasOrg1",
+				  "planType": "Atlas",
+				  "groupId": "5e5fbc29e76c9a4be2ed3d39",
+				  "clusters": [
+					{
+					  "backupEnabled": true,
+					  "authEnabled": true,
+					  "alertCount": 0,
+					  "versions": [
+						"3.4.2"
+					  ],
+					  "sslEnabled": true,
+					  "name": "AtlasCluster1",
+					  "dataSizeBytes": 1000000,
+					  "nodeCount": 7,
+					  "availability": "unavailable",
+					  "clusterId": "5e5fbc29e76c9a4be2ed3d4d",
+					  "type": "sharded cluster"
+					},
+					{
+					  "backupEnabled": false,
+					  "authEnabled": true,
+					  "alertCount": 0,
+					  "versions": [
+						"3.4.1"
+					  ],
+					  "sslEnabled": true,
+					  "name": "AtlasReplSet1",
+					  "dataSizeBytes": 1300000,
+					  "nodeCount": 2,
+					  "availability": "dead",
+					  "clusterId": "5e5fbc29e76c9a4be2ed3d4f",
+					  "type": "replica set"
+					}
+				  ],
+				  "orgId": "5e5fbc29e76c9a4be2ed3d36"
+				},
+				{
+				  "groupName": "CloudGroup1",
+				  "orgName": "TestCloudOrg1",
+				  "planType": "Cloud Manager",
+				  "groupId": "5e5fbc29e76c9a4be2ed3d38",
+				  "clusters": [
+					{
+					  "backupEnabled": true,
+					  "authEnabled": false,
+					  "alertCount": 0,
+					  "versions": [
+						"3.4.1",
+						"2.4.3"
+					  ],
+					  "sslEnabled": false,
+					  "name": "cluster1",
+					  "dataSizeBytes": 1000000,
+					  "nodeCount": 6,
+					  "availability": "warning",
+					  "clusterId": "5e5fbc29e76c9a4be2ed3d42",
+					  "type": "sharded cluster"
+					},
+					{
+					  "backupEnabled": true,
+					  "authEnabled": true,
+					  "alertCount": 0,
+					  "versions": [
+						"3.4.1"
+					  ],
+					  "sslEnabled": true,
+					  "name": "replica_set",
+					  "dataSizeBytes": 500000,
+					  "nodeCount": 2,
+					  "availability": "available",
+					  "clusterId": "5e5fbc29e76c9a4be2ed3d3c",
+					  "type": "replica set"
+					},
+					{
+					  "backupEnabled": null,
```

Comment:
null value for the bool field

Reply:
Where did you get this value from? when I tested the API I got false, here's an example response from my machine
OM
```json
{
  "links" : [ {
    "href" : "http://mms:8080/api/public/v1.0/clusters?pretty=true&envelope=false",
    "rel" : "self"
  } ],
  "results" : [ {
    "clusters" : [ {
      "alertCount" : 0,
      "authEnabled" : false,
      "availability" : "available",
      "backupEnabled" : false,
      "clusterId" : "x",
      "dataSizeBytes" : 0,
      "name" : "myReplicaSet",
      "nodeCount" : 3,
      "sslEnabled" : false,
      "type" : "replica set",
      "versions" : [ "4.2.3" ]
    } ],
    "groupId" : "x",
    "groupName" : "Project 0",
    "orgId" : "x",
    "orgName" : "Organization 0",
    "planType" : "Ops Manager",
    "tags" : [ ]
  } ],
  "totalCount" : 1
}
```

Atlas
```json
{
  "links" : [ {
    "href" : "http://atlas/api/atlas/v1.0/clusters?pretty=true",
    "rel" : "self"
  } ],
  "results" : [ {
    "clusters" : [ {
      "alertCount" : 0,
      "authEnabled" : true,
      "availability" : "dead",
      "backupEnabled" : false,
      "clusterId" : "x",
      "dataSizeBytes" : 0,
      "name" : "Cluster0",
      "nodeCount" : 0,
      "sslEnabled" : true,
      "type" : "replica set",
      "versions" : [ "4.2.3" ]
    } ],
    "groupId" : "x",
    "groupName" : "Test",
    "orgId" : "x",
    "orgName" : "Atlas",
    "planType" : "Atlas",
    "tags" : [ ]
  } ],
  "totalCount" : 1
}
(I removed ids from here just in case)

Reply:
That is the value that I got when I run the api test from om